### PR TITLE
Tweak the global metadata key design.

### PR DESCRIPTION
**Pull request by petermattis**
Provide support for looking up a table descriptor given a table ID.

### CODE REVIEW

File: docs/structured.md [hunk 1]
```diff
@@ -92,22 +92,42 @@ be taken to not expose these implementation details to the user.
...
 
-`\x00ns<NamespaceName>` : `NamespaceDescriptor`
+`\x00ns-<ParentID><Name>` : `<DescriptorID>`
 
-`\x00tbl<NamespaceID><TableName>` : `TableDescriptor`
+`\x00desc-<DescriptorID>` : `{Namespace,Table}Descriptor`
+
+The root namespace has the fixed ID of `1`. The initial implementation will
+support only a single level of namespaces, though the key design allows for
+arbitrary many levels. The namespace table, mapping name to descriptor ID,
```
Comment (tamird) [inline docs/structured.md]: arbitrarily

Comment (petermattis) [inline docs/structured.md]: Yes.

**Comment by tamird** [comment]
LGTM

File: docs/structured.md [hunk 2]
```diff
@@ -92,22 +92,42 @@ be taken to not expose these implementation details to the user.
...
+form the key. The actual code does not use a "/" to separate key
+components, but instead uses the encodings founding in
+[util/encoding](https://github.com/cockroachdb/cockroach/tree/master/util/encoding).
 
 ###Global metadata key###
 
 The metadata addressing will use the following key:Descriptor mappings:
 
-`\x00ns<NamespaceName>` : `NamespaceDescriptor`
+`\x00ns-<ParentID><Name>` : `<DescriptorID>`
```
Comment (vivekmenezes) [inline docs/structured.md]: s/ParentID/Parent-DescriptorID

**Comment by vivekmenezes** [comment]
LGTM

File: docs/structured.md (hunk 2 above)
Comment (petermattis) [inline docs/structured.md]: Ok.

**Pull request merged**